rubin_sim/maf/metricBundles/moMetricBundle.py

```diff
@@ -86,6 +86,8 @@ def makeCompletenessBundle(bundle, completenessMetric, Hmark=None, resultsDb=Non
         mb.setSummaryMetrics(metric)
         mb.computeSummaryStats(resultsDb)
         val = mb.summaryValues["Value At H=%.1f" % Hmark]
+        if val is None:
```

Comment:
I had to add this here to get things to run. I think it's just a case where running on a very small example survey there can be no detections of a population, and thus the summaryValue ends up as a `None` which will fail later when it expects a float. But it would be good to double check this isn't actually a bug upstream.

Reply:
Can you provide an example case where it fails? I don't think I've seen a problem, so it may indeed be only for odd cases with no detections. Reasonable to catch it though.